tests/click_tests/test_passphrase_bde.py

```diff
@@ -114,6 +116,22 @@ def go_to_category(debug: "DebugLink", category: PassphraseCategory) -> None:
     else:
         for _ in range(current_index - target_index):
             debug.swipe_right()
+    if check:
+        # map Rust debug string to Python enum
+        layout_map = {
+            "LettersLower": PassphraseCategory.LOWERCASE,
```

Comment:
I know this is unrelated to this PR, but why not rename either one or the other enum members to have them match?